packages/common/log.js

```diff
@@ -7,7 +7,10 @@ const isString = require('lodash.isstring');
 function logger() {
   return new Logger({
     executions: process.env.EXECUTIONS,
+    granules: process.env.GRANULES,
```

Comment:
Where are these environment variables being set?

Reply:
they're being set in the message adapter client js . https://github.com/nasa/cumulus-message-adapter-js/pull/28

It's optional, fo if it's not set, it just doesn't show up.